### PR TITLE
Fix/supress ReSharper warnings for Microsoft.EntityFrameworkCore.SqlServer

### PR DESCRIPTION
**Pull request by AndriySvyryd**

### CODE REVIEW

**Comment by anpete** [comment]
:shipit:

File: src/Microsoft.EntityFrameworkCore.Commands/Design/DbContextOperations.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public virtual Type GetContextType([CanBeNull] string name)
             var contexts = new Dictionary<Type, Func<DbContext>>();
 
             // Look for IDbContextFactory implementations
-            var contextFactories = _startupAssembly.GetConstructibleTypes()
+            var contextFactories = _startupAssembly.GetConstructableTypes()
```
Comment (lajones) [inline src/Microsoft.EntityFrameworkCore.Commands/Design/DbContextOperations.cs]: Weird that R# insists on this. AFAICS "constructible" is more normal, but if it stops R# warnings then so be it.

**Comment by lajones** [comment]
:shipit:

File: src/Microsoft.EntityFrameworkCore.Commands/Design/DbContextOperations.cs (hunk 1 above)
Comment (AndriySvyryd) [inline src/Microsoft.EntityFrameworkCore.Commands/Design/DbContextOperations.cs]: The dictionary it's using is not very flexible, e.g. It also marks behaviour as misspelled

**Pull request merged**